lib/src/card/Card.tsx

```diff
@@ -55,25 +53,25 @@ const DxcCard = ({
   );
 };
 
-const StyledDxcCard = styled.div`
+const StyledDxcCard = styled.div<CardPropsType & { hasAction: (() => void) | string }>`
```

Comment:
I don't think there's a need in using CardPropsType. It can be more readable to have only the necessary types.